pkg/quota/install/registry.go

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
the intention is that this would aggregate a registry from each api group (core, extensions, etc.) supported by the kube-apiserver.